bw2io/strategies/ecospold2.py

```diff
@@ -10,6 +10,60 @@
 
 
 def link_biosphere_by_flow_uuid(db, biosphere="biosphere3"):
+    """
+    Modify the exchanges in the given list of datasets to link to the given
```

Comment:
For consistency: here you use the command or imperative ("Perform XYZ"), whereas below you often use declaratives ("This function performs XYZ").

Reply:
I believe it makes more sense to use the imperative throughout.

Reply:
(I did not add comments to all instances below - please check)

Reply:
sure noted

Reply:
@Cerebrovinny,  I think you did the opposite of what I recommended 😅
I would prefer:

"""Perform action XYZ""" (_imperative_)

instead of 

"""This function performs action XYZ""" (_declarative_)

As soon as this is resolved, I'll merge 😇